### PR TITLE
Initial firestore video sync

### PR DESCRIPTION
**Pull request by clayferg**
If you put an API_KEY var in ApiKey.js file and open multiple windows (I've only tried up to 3, but more should work) from a dev server, you should be able to test the sync out.

### CODE REVIEW

File: src/main/webapp/IFrame.js [hunk 1]
```diff
@@ -12,143 +12,189 @@
...
-    }
-  }
-}
+let catchUp = false; // Does this vid need to catch up to Firestore?
 
-// Get the time of the current video
-function getTime() { // eslint-disable-line no-unused-vars
-  console.log(player.getCurrentTime());
-}
+/*
```
Comment (krowngold) [inline src/main/webapp/IFrame.js:65]: Could we maybe shorten this comment? Seems like we're giving up some implementation details that may not be necessary

Comment (clayferg) [reply src/main/webapp/IFrame.js:65]: Sure thing!

File: src/main/webapp/IFrame.js [hunk 2]
```diff
@@ -12,143 +12,163 @@
...
+const SYNC_WINDOW = 5; // max time diff between client and Firestore
+
+const firebaseConfig = {
+  apiKey: API_KEY, // eslint-disable-line no-undef
+  authDomain: 'lounge-95f01.firebaseapp.com',
+  databaseURL: 'https://lounge-95f01.firebaseio.com',
+  projectId: 'youtube-lounge',
+  storageBucket: 'youtube-lounge.appspot.com',
+  messagingSenderId: '681171972170',
+  appId: '1:681171972170:web:4c6526b8eb788af9d876b3',
```
Comment (ciddu) [inline src/main/webapp/IFrame.js:25]: Is any of this unique to the app and be considered private information?

Comment (clayferg) [reply src/main/webapp/IFrame.js:25]: Yeah, I'll move it to a private file.

**Pull request merged**